content/guides/getting-started.md

```diff
@@ -109,7 +109,7 @@ __src/index.js__
 +
   function component() {
     var element = document.createElement('div');
-  
+
 -   // Lodash, currently included via a script, is required for this line to work
```

Comment:
Can we leave this comment? It's to illustrate that in the basic example we explicitly depend on lodash.